tests/conftest.py

```diff
@@ -64,4 +64,4 @@ def s3_client(mocker):
     """
     mocks boto client
     """
-    return mocker.patch('boto3.client', new_callable=mocks.s3ClientMock)
+    return mocker.patch('airflow.hooks.S3_hook.S3Hook.get_conn', new_callable=mocks.s3ClientMock)
```

Comment:
shouldn't we mock `get_s3_client`?

Reply:
if it works good, if it doesn't don't do it

Reply:
This is more complicated than it would seem due to python importing.